grails-app/services/nl/thehyve/gb/backend/user/UserService.groovy

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
maybe we need to pick this one https://jira.thehyve.nl/browse/TMT-261 at some point and reuse the library here